Web/com/metacube/db/helper/EmployeeDBHelper.java

```diff
@@ -0,0 +1,114 @@
+package com.metacube.db.helper;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.metacube.db.ConnectionFactory;
+import com.metacube.model.Employee;
+
+public class EmployeeDBHelper {
+
+    private static final String GET_EMPLOYEE_LIST_QUERY = "SELECT * FROM Employee";
+    private static final String GET_EMPLOYEE_BY_ID_QUERY = "SELECT * FROM Employee where id = ?";
+    private static final String CREATE_EMPLOYEE_QUERY = "INSERT INTO employee(name, email, address) VALUES(?, ?, ?)";
+    private static final String GET_EMPLOYEE_BY_NAME_QUERY = "SELECT * FROM Employee WHERE name = ?";
+    private static final String DELETE_EMPLOYEE_BY_ID_QUERY = "delete from Employee where id = ?";
+
+    public int createEmployee(Employee employee) {
+	int created = -1;
+	try {
+	    Connection connection = ConnectionFactory.getConnection();
+	    PreparedStatement preparedStatement = connection.prepareStatement(CREATE_EMPLOYEE_QUERY);
+	    preparedStatement.setString(1, employee.getName());
+	    preparedStatement.setString(2, employee.getEmail());
+	    preparedStatement.setString(3, employee.getAddress());
+	    created = preparedStatement.executeUpdate();
+	    System.out.println(created);
+	    connection.commit();
```

Comment:
By Default, setAutoCommit() property of Connection is set to true. So no need to put commit(). Use commit() and rollback() method when you explicitly put the setAutoCommit() to false.